### ISSUE TITLE
Information provided on hover loss

### ISSUE THREAD

**Issue by ydmykr** [microsoft/pylance-release#4571]
![image](https://github.com/microsoft/pylance-release/assets/100815370/bf275540-4594-4752-a658-7e324bb187ab)

![image](https://github.com/microsoft/pylance-release/assets/100815370/5e38a476-f808-4f38-948b-c3d8dfa0885c)

![image](https://github.com/microsoft/pylance-release/assets/100815370/b06d845d-e057-4e89-90c6-14d4edd6bab8)

![image](https://github.com/microsoft/pylance-release/assets/100815370/8930494c-bb66-47d6-84c4-ae229f437d99)

I found in a teaching video two years ago that my hover window displayed differently from the video.

I tried using jedi and pylance, but the document content in the hover window was lost. What is the reason for this?

**Comment by debonte** [comment]
Am I understanding correctly that with the screenshots above you're showing that you do see the docstring for `print` but not for `torch.optim.SGD`? Where is the third screenshot from? The one that does show the `SGD` docstring?

For what it's worth, after installing pytorch via `pip3 install torch torchvision torchaudio`, I don't see a docstring on the implementation (`.venv\Lib\site-packages\torch\optim\sgd.py`) or stub (`.venv\Lib\site-packages\torch\optim\sgd.pyi`) of `SGD`, so that's why we're not showing any docs.

**Comment by ydmykr** [comment]
@debonte Thanks for your response. The first and third figures come from the video. Maybe there's something changed to the package because this is a video without timeliness.
